config/routes.rb

```diff
@@ -4,10 +4,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  namespace :api do
+  namespace :api, defaults: { format: :json } do
     namespace :v1 do
-      resources :users
-      post 'auth/login/', to: 'authentication#login'
```

Comment:
[ Required ] Hi @henrycode460, I think these routes are important for `User` registration and `User` login so we should not delete them. Please consider adding them back to the `routes.rb` file